app/models/github_fetcher/issues_response.rb

```diff
@@ -0,0 +1,21 @@
+module GithubFetcher
+  class IssuesResponse
```

Comment:
the nice thing about a method like this is that it would be super easy to stub in tests...